src/js/services/ws.cy.js

```diff
@@ -89,24 +89,24 @@
   specify('Restarting a closed socket', function() {
     const channel = Radio.channel('ws');
 
-    cy.stub(service, 'restart').as('restart');
+    const closedTest = { name: 'SendTest', data: 'CLOSED' };
 
-    // Start service
-    channel.request('send', { name: 'SendTest', data: 'OPENED' });
+    cy
+      .startService()
+      .then(() => {
+        cy.stub(service, 'start').as('start');
 
-    service.on('start', () => {
-      // websocket is closed
-      service.ws.close();
+        service.ws.addEventListener('close', () => {
+          channel.request('send', closedTest);
+        });
 
-      // send after close
-      service.ws.onclose = () => {
-        channel.request('send', { name: 'SendTest', data: 'CLOSED' });
-      };
-    });
+        service.ws.close();
+      });
 
     cy
-      .get('@restart')
-      .should('have.been.calledOnce');
+      .get('@start')
+      .should('have.been.calledOnce')
+      .and('have.been.calledWith', { state: {}, data: closedTest });
```

Comment:
this doesn't seem correct.. stubbing it for when it is called during the tests.. wrapping it at this point.. it's already called.. I don't see how this would do anything